### PR TITLE
:sparkles: Add an option to enable sharing on Workplace

### PR DESCRIPTION
**Pull request by ClementVanPeuter**
See #156

### CODE REVIEW

File: server/prisma/datamodel.graphql [hunk 1]
```diff
@@ -127,4 +127,6 @@ type Configuration {
   slackChannelHook: String
 
   tags: Json
+
+  workplaceButton: Boolean! @default(value:false)
```
Comment (hgwood) [inline server/prisma/datamodel.graphql]: `enableWorkplaceSharing` or `workplaceSharingEnabled` would be clearer I think.

Comment (ClementVanPeuter) [reply server/prisma/datamodel.graphql]: Good

**Comment by Errorname** [comment]
Maybe we should add a rule to remove the generated prisma graphql from the linter. This isn't useful and it breaks the linting every time we re-generate it.

Can you add this rule in your PR?

**Comment by hgwood** [comment]
@ClementVanPeuter sorry I misunderstood what you meant when we talked about this. I took the liberty to do what @Errorname asked.

@Errorname I made prettier ignore everything in src/generated. Is that alright?

**Comment by Errorname** [comment]
@hgwood Yes, that's perfect.

**Comment by nlepage** [comment]
Oups :grin: 
That's interesting information, thank you z-lookout-bot...

**Comment by Bugsyaya** [comment]
Result :
capture d ecran de 2019-02-14 15-28-46

**Comment by Errorname** [comment]
Great work team 💪

**Pull request merged**